### ISSUE TITLE
[vis-network][DOMtoCanvas]After the canvas zoom in or out, node coordinate transformation

### ISSUE THREAD

**Issue by ritevke** [almende/vis#3427]
when the user zooms in or out，dragged into a node created from the outside, use of the container DOM coordinates into Canvas coordinates, there is a deviation?Canvas during initialization, external to transform the coordinates of nodes can be normal.

$scope.canvasToDOM = $scope.visNetwork.DOMtoCanvas({x:event.offsetX,y:event.offsetY});

**Comment by ritevke** [comment]
感觉我说的你们可能不太明白，这个问题大概的意思是这样的，当初始化画布时，我的容器坐标用DOMtoCanvas来转是正确获取到canvasToDOM ，但如果画布被用户放大或缩小后，再来转换坐标的位置就不对了，是代码的本身有bug，还是要怎么去转换，期待回复！！谢谢！！！

**Comment by ritevke** [comment]
Who can help me?

**Comment by wimrijnders** [comment]
@ritevke truly sorry about lack of response. But I'm really busy, not just outside `vis.js` but with fixing critical bugs within. I'm really hoping to get around to it, or that someone else may help you.

**Comment by ritevke** [comment]
@wimrijnders I'm very sorry, but this problem really troubled for a long time.
Simply speaking, when the canvas is initialized, the coordinate conversion is fine, but the canvas is transformed into a wrong coordinate after it is enlarged or shrunk. I wonder if you can understand.

**Comment by wimrijnders** [comment]
Yes I think understand. I'll check on my setup if the same happens.

**Comment by wimrijnders** [comment]
Sorry, I can't reproduce. I can drag and zoom all I want, and the coordinates, DOM or Canvas, are completely as expected.

I did the following to test: I took the [`interactionEvents` example](http://visjs.org/examples/network/events/interactionEvents.html) and adjusted the `click`-handler to the following:

```js
 network.on("click", function (params) {
        // This block added
        var tmpEvent = params.event.changedPointers[0];
        var toCanvas = network.DOMtoCanvas({x:tmpEvent.offsetX, y:tmpEvent.offsetY});
        console.log('event offsetXY: ' + tmpEvent.offsetX + ', ' + tmpEvent.offsetY);
        console.log(toCanvas);
        // End added

        params.event = "[original event]";
        document.getElementById('eventSpan').innerHTML = '<h2>Click event:</h2>' + JSON.stringify(params, null, 4);
        console.log('click event, getNodeAt returns: ' + this.getNodeAt(params.pointer.DOM));
    });

```
 ... and then compared the values displayed in the console with the internal values. These match *exactly*.

Perhaps I'm missing something in this code fragment. Feel free to suggest changes to better demonstrate the issue.
